architecture/core_objects/builds.adoc

```diff
@@ -223,9 +222,9 @@ that you created, which in the above example is `*dockerhub*`:
 ----
 ====
 
-Pull the builder Docker image from a private Docker registry by specifying the
+. Pull the Docker build image from a private Docker registry by specifying the
 `PullSecret` field, which is part of the build strategy definition:
-
++
```

Comment:
Yes, this is part of the BuildConfig specification, this part defines the strategy used during build.